modules/cryoGridTechnical/sum_up_output_store.m

```diff
@@ -1,191 +1,211 @@
+    OUT.TIMESTEP=[OUT.TIMESTEP; TEMPORARY.timestep_out];
+    OUT.timestamp=[OUT.timestamp; t];
+    
+    % realated to surface energy balance
+    OUT.SEB.Lsta=[OUT.SEB.Lsta; mean(SEB.L_star)];
+    OUT.SEB.QE=[OUT.SEB.QE; TEMPORARY.Qe];
+    OUT.SEB.QH=[OUT.SEB.QH; TEMPORARY.Qh];
+    OUT.SEB.QG=[OUT.SEB.QG; TEMPORARY.Qg];
+    OUT.SEB.QNET=[OUT.SEB.QNET; TEMPORARY.Qnet];
+    OUT.SEB.Tsurf=[OUT.SEB.Tsurf; TEMPORARY.T_out(GRID.air.cT_domain_lb+1)];
+    OUT.SEB.albedo_stored=[OUT.SEB.albedo_stored; PARA.surf.albedo];
+    
+    
+    % related to soil
+    OUT.soil.topPosition=[OUT.soil.topPosition; -GRID.general.K_grid(GRID.soil.cT_domain_ub)];
+    if ~isempty( GRID.lake.cT_domain_ub )
+        OUT.soil.lakeFloor = [OUT.soil.lakeFloor; - GRID.general.K_grid(GRID.lake.cT_domain_lb+1) ];
+    else
+        OUT.soil.lakeFloor = [OUT.soil.lakeFloor; NaN];
+    end
+    OUT.soil.soil{1, size(OUT.soil.soil,2)+1}=[GRID.soil.cT_water GRID.soil.cT_mineral GRID.soil.cT_organic];
+    
+    
+    % related to snow
+    OUT.snow.outSnow_i=[OUT.snow.outSnow_i GRID.snow.Snow_i];
+    OUT.snow.outSnow_a=[OUT.snow.outSnow_a GRID.snow.Snow_a];
+    OUT.snow.outSnow_w=[OUT.snow.outSnow_w GRID.snow.Snow_w];
+    if ~isempty(GRID.snow.cT_domain_ub)
+        TEMPORARY.topPosition=-GRID.general.K_grid(GRID.snow.cT_domain_ub);
+        TEMPORARY.botPosition=-GRID.general.K_grid(GRID.snow.cT_domain_lb+1);
+    else
+        TEMPORARY.topPosition=NaN;
+        TEMPORARY.botPosition=NaN;
+    end
+    OUT.snow.topPosition=[OUT.snow.topPosition; TEMPORARY.topPosition];
+    OUT.snow.botPosition=[OUT.snow.botPosition; TEMPORARY.botPosition];
+    
+    % derived characteristics and related to geometry
+    OUT.location.area = [OUT.location.area; PARA.location.area];
+    OUT.location.altitude=[OUT.location.altitude; PARA.location.altitude];
+    OUT.location.surface_altitude=[OUT.location.surface_altitude; PARA.location.surface_altitude];
+    OUT.location.active_layer_depth_altitude = [OUT.location.active_layer_depth_altitude; PARA.location.active_layer_depth_altitude];
+    OUT.location.water_table_altitude=[OUT.location.water_table_altitude; PARA.location.water_table_altitude];
+    
+    OUT.location.active_layer_depth_altitude_mean = [OUT.location.active_layer_depth_altitude_mean ; TEMPORARY.bottomBucketSoilDepth];
+    OUT.location.water_table_altitude_mean = [OUT.location.water_table_altitude_mean ; TEMPORARY.waterTableElevation];
+    
+    % lateral fluxes
+    if PARA.modules.lateral==1;
         OUT.lateral.terrain_index_snow=[ OUT.lateral.terrain_index_snow; PARA.ensemble.terrain_index_snow ];
-        OUT.lateral.water_fluxes = [ OUT.lateral.water_fluxes; water_fluxes ];     % vector containing water fluxes in [m/s] to the current worker
+        OUT.lateral.water_fluxes = cat(3,OUT.lateral.water_fluxes, BALANCE.water.dr_water_fluxes_out );     % vector containing water fluxes in [m/s] to the current worker

```

Comment:
Is it a vector or a matrix?

For consistency reasons with the snow_fluxes and heat_fluxes, it would be nice to have a vector containig the fluxes (in m/s) to/from each worker here.